2016/figures.html

```diff
@@ -0,0 +1,190 @@
+</header>
+
+<div class="section section--white">
+  <div class="stat-container">
+
+    <div class="stat">
+      <p class="stat__main">
+        <span class="stat__number">26</span>
+        <span class="stat__name">mySociety team members</span>
+      </p>
+      <p class="stat__comment">
+        Two fresh new <strong>developers</strong>, a sparkling <strong>sysadmin</strong>, and a splendid <strong>research associate</strong> joined <a href="https://www.mysociety.org/about/team">the team</a> in 2016.
+      </p>
+    </div>
+
+    <div class="stat">
+      <p class="stat__main">
+        <span class="stat__number">7</span>
+        <span class="stat__name">new board members</span>
+      </p>
+      <p class="stat__comment">
+        Phew! Thanks to some expansion at the top, we can call on the expertise of <a href="https://www.mysociety.org/about/board/">4 new trustees</a>, and 3 new <a href="https://www.mysociety.org/about/board/">non-executive directors</a> to help plot mySociety’s course.
+      </p>
+    </div>
+
+  </div>
+</div>
+
+<div class="section section--white">
+  <img src="{{ site.baseurl }}/assets/img/quote-positive-difference.png" srcset="{{ site.baseurl }}/assets/img/quote-positive-difference@2x.png 2x" alt="What do you like best about working at mySociety? Louise (Senior Developer): I feel like what we do makes a positive difference in the wider world." class="quote quote--positive-difference">
+
+  <div class="stat-container">
+
+    <div class="stat">
+      <p class="stat__main">
+        <span class="stat__number">[?????]</span>
+        <span class="stat__name">code commits</span>
+      </p>
+      <p class="stat__comment">
+        Every time we press the button marked ‘commit’, we’re making changes and additions to our code. This figure’s wildly up on last year’s (15,723) because of the diligent little EveryPolitician Bot. Find out more about that later.
+      </p>
+    </div>
+
+    <div class="stat">
+      <p class="stat__main">
+        <span class="stat__number">[?????]</span>
+        <span class="stat__name">tickets closed</span>
+      </p>
+      <p class="stat__comment">
+        If you report a problem or suggest a feature, we open a ticket. Then we fix things. ‘Tickets closed’ is one highly satisfying measure of improvement.
+      </p>
+    </div>
+
+  </div>
+</div>
+
+<div class="section section--figures-partners">
+
+  <div class="stat-container">
+
+    <div class="stat">
+      <p class="stat__main">
+        <span class="stat__number">58</span>
+      </p>
+      <p class="stat__comment">
+        Applications from prospective partners around the world
+      </p>
+    </div>
+
+    <div class="stat">
+      <p class="stat__main">
+        <span class="stat__number">10</span>
+      </p>
+      <p class="stat__comment">
+        New partner sites around the world. From Liberia to Costa Rica, and plenty of countries between.
```

Comment:
Change to "New partner sites. From Liberia..." 
- 'around the world' occurs rather a lot in this section.